tests/test_hierarchy.cc

```diff
@@ -324,6 +325,23 @@ BOOST_DATA_TEST_CASE(
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "None"                  , "lanczos" , "matrix-free")] = 0.2990825376;
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "Reverse Cuthill_McKee" , "lanczos" , "matrix-full")] = 0.1024523994;
     ref_solution[std::make_tuple("hyper_ball" , "distort"    , "Reverse Cuthill_McKee" , "lanczos" , "matrix-free")] = 0.2990825376;
+
+    ref_solution[std::make_tuple("hyper_cube" , "no_distort" , "None"                  , "anasazi" , "matrix-full")] = 0.0219765243;
```

Comment:
Are these numbers close to the ones from the other eigensolvers?

Reply:
Yes, really close, if you look up.